modules/shared/src/main/scala/io/constellationnetwork/currency/dataApplication/package.scala

```diff
@@ -417,6 +455,11 @@ object BaseDataApplicationL0Service {
 
       def onGlobalSnapshotPull(snapshot: Hashed[GlobalIncrementalSnapshot], context: GlobalSnapshotInfo): F[Unit] =
         service.onGlobalSnapshotPull(snapshot, context)
+
+      override def getTokenUnlocks(
```

Comment:
I think you implement it here the first time, so you override nothing 🤔

Reply:
👍🏼